test/functional/test_framework/test_node.py

```diff
@@ -186,6 +197,9 @@ def start(self, extra_args=None, *, stdout=None, stderr=None, **kwargs):
         self.running = True
         self.log.debug("bitcoind started, waiting for RPC to come up")
 
+        if self.start_perf:
+            self._start_perf("")
```

Comment:
I think it'd be better to pass `None` here (or even better have `None` as the default argument). `""` and `None` will get treated the same in `(profile_name or 'test')`.

EDIT: I see you're using `profile_name` as the key in the `perf_subprocesses` dictionary so this wouldn't work. Is `perf_subprocesses` safe from having entries overwritten if the same `profile_name` is passed in twice?